include/envoy/network/BUILD

```diff
@@ -10,7 +10,10 @@ envoy_package()
 
 envoy_cc_library(
     name = "address_interface",
-    hdrs = ["address.h"],
+    hdrs = [
+        "address.h",
+        "io_handle.h"
```

Comment:
I'd encourage splitting io_handle.h into its own library and having the relevant libraries depend on it.

Reply:
+1